src/main/java/io/reactivesocket/Frame.java

```diff
@@ -321,6 +323,31 @@ public static String dataMimeType(final Frame frame)
     public static class Error
     {
 
+        private static final Function<Throwable, ByteBuffer> DEFAULT_ERROR_MESSAGE_TRANSFORMER;
+
+        // TODO Make this plugin, and also make it so that a error message transformer can be
+        // TODO passed in with an application
+        static {
+            if (Boolean.getBoolean("io.reactivesocket.debugError")) {
+                DEFAULT_ERROR_MESSAGE_TRANSFORMER = throwable -> {
+                    StringWriter writer = new StringWriter();
+                    PrintWriter printWriter = new PrintWriter(writer);
+                    throwable.printStackTrace(printWriter);
+                    String data = writer.toString();
```

Comment:
I would prepend `throwable.getMessage() == null ? "" : throwable.getMessage();` to the `data` String (sperated by new line) so that in both cases the data starts by the same thing.